cli/ballerina-cli/src/main/resources/cli-help/ballerina-persist.help

```diff
@@ -5,20 +5,18 @@ SYNOPSIS
        bal persist <command> [<-h> | <--help>]
 
 DESCRIPTION
-       Manage data persistence in a Ballerina package.
+       Manage the data persistence in a Ballerina package.
 
-       Initializing the package for persistence is the first step towards achieving
+       Initializing the package for persistence is the first step toward achieving
        data persistence in a Ballerina package. This step creates the required
-       directories and configuration files.
-       For example:
+       directories and configuration files. For example:
 
-           $ bal persist init
+           $ bal persist add
 
-       Defining the application's data model is the next step. Use the generated
-       definition file in the `persist` directory to define the data model
+       Define the application's data model as the next step. Use the generated
+       definition file in the "persist" directory to define the data model
```

Comment:
```suggestion
       definition file in the `persist` directory to define the data model
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/42542/commits/10e8107d9a0e35db389b4d748ba1b8effc819468